### PR TITLE
DOP-1408 - Tooter border issue

### PR DESCRIPTION
**Pull request by ck-makingsense**
Fix: visual issue at footer border

create list test

domain selector

file import

html editor

lultiple list

single list

### CODE REVIEW

File: src/directives/automation/editor/dpEditorListsGrid.js [hunk 1]
```diff
@@ -96,59 +93,62 @@
...
-        var newComponentData;
-        var oldComponentData;
-        var path;
         var selectedComponent = selectedElementsService.getSelectedComponent();
-
-        if (selectedComponent.type === COMPONENT_TYPE.ACTION) {
-          path = 'operation.suscriptionList';
-          data = {
-            operation: {
+        // TODO: Analize else branch
```
Comment (andresmoschini) [inline src/directives/automation/editor/dpEditorListsGrid.js:101]: Is this TODO still pending?

File: src/directives/automation/editor/dpEditorListsGrid.js [hunk 2]
```diff
@@ -6,7 +6,6 @@
     .directive('dpEditorListsGrid', dpEditorListsGrid);
 
   dpEditorListsGrid.$inject = [
-    '$rootScope',
```
Comment (andresmoschini) [inline src/directives/automation/editor/dpEditorListsGrid.js:9]: I do not understand the change completely, but removing dependence on the `rootScope` and changing the event-based code to simple functions looks great!

File: src/directives/automation/editor/dpEditorListsGrid.js (hunk 1 above)
Comment (ck-makingsense) [reply src/directives/automation/editor/dpEditorListsGrid.js:101]: yes that is right, the goal is resolve the visual issue in this PR, and not improve the logic. I could detect the else branch is not used, with a lazy test

Comment (andresmoschini) [reply src/directives/automation/editor/dpEditorListsGrid.js:101]: ✔️

File: src/directives/automation/editor/dpEditorListsGrid.js (hunk 2 above)
Comment (ck-makingsense) [reply src/directives/automation/editor/dpEditorListsGrid.js:9]: rootScope, is not used and it was added without needed

Comment (andresmoschini) [reply src/directives/automation/editor/dpEditorListsGrid.js:9]: Oh, in fact, it was used:

image

And, for that reason, your change looks better yet.

Comment (ck-makingsense) [reply src/directives/automation/editor/dpEditorListsGrid.js:9]: jaja, you right. I saw rootscope with light text after refactor

**Pull request merged**